.github/workflows/webhook.yml

```diff
@@ -0,0 +1,17 @@
+name: Discord Webhook
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  send-payload:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Discord notification
+      env:
```

Comment:
Changed the name of the job to "commit-stream", since that's what it's generally referred to, and is what the Discord is named as well, the default step name seems fine so I've left it blank